Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -347,7 +379,7 @@ extension PackageController {
         let baseURL = "http://\(baseURLHost)/\(baseURLPath)"
 
         switch fragment {
-            case .css, .data, .documentation, .images, .index, .js:
+            case .css, .data, .documentationRedirect, .documentation, .images, .index, .js:
                 return URI(string: "\(baseURL)/\(fragment)/\(path)")
```

Comment:
I'm not sure this is actually working as intended. This is generating the following urls in AWS which would definitely not be working:

```
(lldb) po URI(string: "\(baseURL)/\(fragment)/\(path)")
▿ http://awsDocsBucket.s3-website.us-east-2.amazonaws.com/owner/package/1.2.3/documentationRedirect/
  - string : "http://awsDocsBucket.s3-website.us-east-2.amazonaws.com/owner/package/1.2.3/documentationRedirect/"
```